### DISCUSSION TITLE
print preview

### DISCUSSION THREAD

**Discussion by Galaeasy013274** [mriscoc/Ender3V2S1#687] [General]
Hi, in the Ender3V2S1-20221222 firmware after about ten prints the print preview no longer works, I also tried deleting the script for cure and put it back but it doesn't work, I don't know if it's a cure or firmware problem.

**Comment by mriscoc** [comment]
Test the preview in your printer with this file: https://github.com/mriscoc/Ender3V2S1/raw/Ender3V2S1-Released/slicer%20scripts/cura/SimpleCuraTest.gcode

**Reply by mriscoc** [reply]
If it doesn't work, your current board firmware/display firmware is unable to show previews.

**Comment by Galaeasy013274** [comment]
it does not work

**Reply by mriscoc** [reply]
What screen do you get when you select that file? Post a screen capture.

**Comment by Galaeasy013274** [comment]
il qrcode
